docs/hosting-a-server-with-northstar/dedicated-server/README.md

```diff
@@ -95,6 +95,18 @@ Example: `+setplaylistvaroverrides "run_epilogue 0 featured_mode_amped_tacticals
 | `player_bleedout_firstAidHealPercent`        |                 |               |                                                                                                |
 | `player_bleedout_aiBleedingPlayerMissChance` |                 |               |                                                                                                |
 
+### Sticks and Stones playlist overrides
+
+| PlaylistOverrides for SNS                    | Accepted Values | Default Value | Description                                                                                    |
+| -------------------------------------------- | --------------- | ------------- | ---------------------------------------------------------------------------------------------- |
+| sns_softball_enabled                       | 0-1           | 0           | Enables Softball usage                                                                        |
```

Comment:
Wait, softball is in default sticks&stones but behind a convar? o_O

I always assumed servers that had it used an extra mod ^^

Reply:
Yes, softball is hidden behind a convar lol. It makes the game mode exponentialally more fun allowing you to rocket/grenade jump/slide with it. I'm currently making changes to the gamemode to make kills more efficient for the server (it looks for a reset kill first wtf) and differentiate execution and pulse blade kills. Have the second one finished but am on vacation so I will be unavailable to work on it until I'm back.

Reply:
Also, later I would like to add the ability and variables to change the starting wait time of a match instead of just outright disabling it (the classic mp intro). Have the variable changed built in code for testing but need to set it up so it can be changed by servers or clients.